README.md

```diff
@@ -1,42 +1,20 @@
-# redux-immutable-state-invariant
+# object-invariant-test-helper
 
-Redux middleware that spits an error on you when you try to mutate your state either inside a dispatch or between dispatches. **For development use only!**
+A module that is useful for when needing to test for immutability.
 
-## Why?
+This module began life as a fork of redux-immutable-state-invariant by Leonardo Andres Garcia Crespo.
```

Comment:
Put a link behind this credit to his GitHub profile? And a link behind the `redux-imm...` to the original project?